src/coreclr/pal/src/misc/cgroup.cpp

```diff
@@ -179,7 +179,7 @@ class CGroup
         char *mountpath = nullptr;
         char *mountroot = nullptr;
 
-        FILE *mountinfofile = fopen(PROC_MOUNTINFO_FILENAME, "r");
+        FILE *mountinfofile = fopen(PROC_MOUNTINFO_FILENAME, "rb");
```

Comment:
The stuff in this file doesn't need to be changed as it is always compiled against the system headers.